includes/discovery/discovery-protocols.inc.php

```diff
@@ -205,23 +205,22 @@
 } elseif ($device['os'] == 'jetstream') {
     echo ' JETSTREAM-LLDP MIB: ';
 
-    $lldp_array = SnmpQuery::hideMib()->walk('TPLINK-LLDPINFO-MIB::lldpNeighborInfoEntry')->table();
+    $lldp = SnmpQuery::hideMib()->walk('TPLINK-LLDPINFO-MIB::lldpNeighborInfoEntry')->table();
 
-    foreach ($lldp_array as $key => $lldp) {
-        if (! is_array($lldp['lldpNeighborPortIndexId'])) {
+    foreach ($lldp['lldpNeighborPortIndexId'] as $IndexId => $lldp_data) {
```

Comment:
May be you could add an `is_array` test on  `$lldp['lldpNeighborPortIndexId']` before the foreach ? to avoid a crash if missing ?

Reply:
Done :)